Cargo.toml

```diff
@@ -33,3 +33,7 @@ winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
 spirv_cross = { git = "https://github.com/servo/spirv_cross", branch = "wgpu-servo" }
 surfman-chains = { git = "https://github.com/asajeffrey/surfman-chains" }
 surfman = { git = "https://github.com/servo/surfman" }
+
+[patch."https://github.com/servo/webrender"]
+webrender = { git = "https://github.com/jdm/webrender/", branch = "serialize" }
```

Comment:
Do these not need to be changed back?

Reply:
Ugh, I think I've been testing the wrong branch :(